challenges/objects-arrays.js

```diff
@@ -29,12 +58,17 @@ console.log(tyrannosaurus.period);
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
 
-let roar = [];
- roar = dinosaurs.map(function(item) {
-    return item.name();
-});
+// let roar = [];
+//  roar = dinosaurs.map(function(item) {
+//     return item.name();
+// });
 
-console.log(tyrannosaurus.roar());
+tyrannosaurus.roar = function() {
+    return "RAWERSRARARWERSARARARRRR!"
+  }
+  
+  console.log(tyrannosaurus.roar());
+  
```

Comment:
```suggestion
tyrannosaurus.roar = function() {
    return "RAWERSRARARWERSARARARRRR!"
  }

  console.log(tyrannosaurus.roar());

``` Awesome this is perfect